01-node-tutorial/answers/10-fs-sync.js

```diff
@@ -0,0 +1,14 @@
+const { readFileSync, writeFileSync } = require('fs');
+console.log('start');
+
+// will read from the elevated folder content
+const first = readFileSync('../content/first.txt', 'utf-8');
+const second = readFileSync('../content/second.txt', 'utf-8');
+
+console.log(first, second);
+
+writeFileSync(
```

Comment:
To fully tests the writeFileSync with append functionality, would be better to split into 3 lines (i.e. writeFile sync for line 1, then call it again for line 2, etc). In other words, closer to how you've done it for the async exercise

That's more to see how append works - the way you've written it here isn't wrong coding wise.

Reply:
Thanks for the feedback on this. Do you mean for me to split it up like this into two separate writeFileSync? 

```
// first write file
writeFileSync(
    '../content/result-sync.txt',
    `Here is the result : ${first}, ${second}`,
    { flag: 'a' }
)

// second write file
writeFileSync(
    '../content/result-sync.txt',
    `Here is the result : ${second}`,
    { flag: 'a' }
)
```

Reply:
Would you like me to merge my pull request if there are no other issues?

Reply:
Yup! You’re good to merge if I approve a PR